backend/api/Controllers/RobotController.cs

```diff
@@ -663,7 +681,15 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
             _logger.LogWarning("Could not find robot with id={id}", scheduleLocalizationMissionQuery.RobotId);
             return NotFound("Robot not found");
         }
-
+        if (robot.Status is RobotStatus.Stuck)
+        {
+            _logger.LogWarning(
+                "Robot '{id}' is stuck ({status})",
+                scheduleLocalizationMissionQuery.RobotId,
+                robot.Status.ToString()
+            );
+            return Conflict($"The Robot is stuck ({robot.Status})");
+        }
```

Comment:
I don't see how this is desired behavior. Should be able to localize even if the robot has some obstacle

Reply:
Agreed - will be removed